### PR TITLE
Spend-before-Sync release candidates

### LINKED ISSUE #860: zcash_primitives 0.13-rc1, zcash_client_backend 0.10-rc1 and zcash_client_sqlite 0.8-rc1

### PR DESCRIPTION
**Pull request by str4d**
Closes #860.

### CODE REVIEW

File: zcash_client_sqlite/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.8.0-rc.1] - 2023-09-08
```
Comment (nuttycom) [inline zcash_client_sqlite/CHANGELOG.md:10]: Need the version bumps in the `Changed` section here also.

**Review by nuttycom — CHANGES_REQUESTED** [review]
`zcash_client_sqlite` changelog needs the version bumps, otherwise LGTM

File: zcash_client_sqlite/CHANGELOG.md (hunk 1 above)
Comment (str4d) [reply zcash_client_sqlite/CHANGELOG.md:10]: They are?

Comment (nuttycom) [reply zcash_client_sqlite/CHANGELOG.md:10]: `zcash_proofs`

Comment (str4d) [reply zcash_client_sqlite/CHANGELOG.md:10]: Oh, I see I missed one: `zcash_proofs` is not a public dependency, so it shouldn't be in the "Changed" section.

Comment (nuttycom) [reply zcash_client_sqlite/CHANGELOG.md:10]: Oh... I guess that dependency is indirect?

**Review by nuttycom — APPROVED** [review]
utACK 62e3299

**Pull request merged**